docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,386 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+
+The default path of the configuration file is `/etc/minicoredumper/minicoredumper.cfg.json` and the file format is JSON. Possible options are:
+
+-   **base_dir** (string): Defines the root directory where the dumped data will be stored. This option is overwritten by the command line option, which is set by **crash-manager**.
+-   **watch** (array): A set of recepts, which will be used for applications that meet the defined criteria:
+    -   **exe** (string): The full path to the binary returned by readlink. `*` character is supported as wildcard. If not specified, the `*` value will be used
+    -   **comm** (string): The basename of the command that was run. `*` character is supported as wildcard. If not specified, the `*` value will be used
+    -   **recept** (string): The full path to the recept file
+
+    > **Note**
+    >
+    > If **exe** and **comm** are specified, then both conditions must be met to use the recept.
+
+Default minicoredumper.cfg.json:
+
+```
+  {
+      "base_dir": "/usr/local/var/crash/minicoredumper",
+      "watch": [
+          {
+              "exe": "/usr/bin/dotnet-launcher",
+              "recept": "dotnet.recept.json"
+          },
+          {
+              "recept": "generic.recept.json"
+          }
+      ]
+  }
+```
+
+
+The default path of the recept files is `/etc/minicoredumper/` and the format is JSON. Possible options are:
+
+-   **stacks** (list): Contains settings for what should be dumped:
+    -   **dump_stack** (bool): If true, stacks will be dumped
+    -   **first_thread_only** (bool): If true, only the first thread will be dumped
+    -   **max_stack_size** (integer): Maximum size of the stack
+-   **maps** (list): Contains list "dump_by_name".
+    -   **dump_by_name** (list): List of pathnames of memory regions that should be dumped (pathnames are read from `/proc/<PID>/maps`)
+-   **compression** (list): Contains settings related to report compression.
+    -   **compressor** (string): Command line of compressor that supports stdin as input and stdout as output (gzip, bzip2, xz)
+    -   **extension**: (string): In case of a compressed core file, the specified value will be appended at the end of the core file name; otherwise, `.compressed` will be appended to the compressed core files
+    -   **in_tar** (bool): Determines whether the core file should be a `.tar` file or not. This is useful because the tar format enables preserving the sparse properties of the core file.
+    -   **dump_auxv_so_list** (bool): If true, the shared object list will be saved in the core file
+    -   **dump_pthread_list** (bool): If true, the pthread list will be saved in the core file
+    -   **dump_robust_mutex_list** (bool): If true then robust mutexes will be saved in the core file
+    -   **dump_scope** (int): Only registered dumps at this or lower value this value will be dumped. This option applies only for libminicoredumper
```

Comment:
As mentioned earlier (https://github.com/Samsung/tizen-docs/pull/791#discussion_r308010193), suggest rephrasing to:

- **dump_scope** (int): Only registered dumps at this value or lower than this value will be dumped. This option applies only to libminicoredumper.